src/lineup/internal/LineUpPanelActions.ts

```diff
@@ -374,6 +394,19 @@ export default class LineUpPanelActions extends EventHandler {
     this.searchBoxProvider.update(items);
   }
 
+  private groupColumnDescs(columnDesc: {text: string, id: string, action: () => void, group: {parent: string, order?: number}}[]): [Map<string, any[]>, ISearchOption[]] {
```

Comment:
Could we extract the interface for `{text: string, id: string, action: () => void, group: {parent: string, order?: number}}` as it is used above too? I suppose we could also use something like `{text: string} & Pick<IAdditionalColumnDesc, 'group'>` to avoid having to redeclare the group option. 

Also, the return value returns `any`, can we return a valid interface here?

Reply:
@dvdanielamoitzi Thanks for the change!